### ISSUE TITLE
Missing cricket-pytest

### ISSUE THREAD

**Issue by lurch** [pybee/cricket#78]
According to the [README](https://github.com/pybee/cricket/blob/master/README.rst), I should be able to do:
```
$ pip install cricket
$ cricket-pytest
```
however that doesn't work :-(  Seems like it only installs the `cricket-django` and `cricket-unittest` commands.
Is this because the version on [PyPI](https://pypi.org/project/cricket/) is so old? (PyPI says it was last updated in June 2016) If so, consider this a request for an updated PyPI release :wink:

**Comment by freakboy3742** [comment]
You're correct that the reason is that the version in master is not the version on PyPI.

The reason we haven't done a release is that the version in master doesn't yet work on all platforms; it involves a transition to use Toga (BeeWare's own widget toolkit), which isn't completely stable on Linux and Windows. We're holding off making a new release until it's stable on other platforms.

**Comment by lurch** [comment]
Haha, that's ironic given that I decided to try cricket after watching https://www.youtube.com/watch?v=yI7NYgP54sw :wink: 

But of course it's entirely understandable, good luck with the transitioning :+1:

**Comment by freakboy3742** [comment]
Closing on the basis that it's a release issue; we should have a new release shortly.
